_build/go.mod

```diff
@@ -4,7 +4,7 @@ module github.com/os-observability/redhat-opentelemetry-collector
 
 go 1.21.0
 
-toolchain go1.21.8
+toolchain go1.22.5
```

Comment:
Is that intended?

Reply:
not really, I think this was my go version :/

Reply:
I'll merge this PR and revert the toolchain version in a follow-up PR when I upgrade to the latest version